src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -39,6 +39,14 @@ export const ShowcaseLayout = ({ CAPI, NAV }: Props) => {
 
     const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
 
+    // Allow 'LayoutSlotBottom' component to render if:
+    // - a 'forceEpic' URL flag is set to true (under any circumstance)
+    // - article not set to hide reader revenue (unless 'forceEpic' is set to true)
+    // TODO: plug 'forceEpic' variable below into the URL param through frontend
+    const forceEpic = false;
+    const renderBottomSlot =
+        forceEpic || (forceEpic && !CAPI.config.shouldHideReaderRevenue);
```

Comment:
I think this logic states you'll never check `!CAPI.config.shouldHideReaderRevenue`?

Reply:
yeah should be `a || ! b`

Reply:
I know this is awkward but there was a reason for it. I was trying to leave this code in such a state that we'd only have to update line `46` later to get the behaviour we need during development (until we're ready to actually start serving the slots on DCR).

I've updated this to only have a boolean that we can set to true or false, until we can read such value from a URL parameter (work in progress separately).